### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by adem4ik**

### CODE REVIEW

File: avidemux_ru.ts [hunk 1]
```diff
@@ -3268,7 +3268,7 @@ Proceed anyway?</source>
     <message>
         <location line="+24"/>
         <source>Unable to set the audio language: No video loaded yet!</source>
-        <translation>Невозможно установить язык аудио: видео не загружено!</translation>
+        <translation>Невозможно установить язык аудио: видео ещё не было загружено!</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: I'd just like to clarify the use case of the function `A_setAudioLang(const char *trackIdxTxt, const char *langueName)` where the source of this translation resides: it is used in command line invocation of Avidemux and "No video loaded yet!" applies to the order of options on the command line, i.e. `--load` must precede `--set-audio-language`. There is no other scenario. Especially having a video loaded at some earlier point in time is not directly related to being able to set audio language: a video (with an active audio track) must be loaded _by_ the moment when A_setAudioLang() is called.

It seems to me that the existing translation matches the actual use case better than the proposed update.

File: avidemux_ru.ts [hunk 2]
```diff
@@ -2767,7 +2767,7 @@ Is it running ?</source>
     <message>
         <location line="+4"/>
         <source>Enable OpenGL display in filter preview dialogs</source>
-        <translation type="unfinished"></translation>
+        <translation>Включить отображение OpenGL в окне предпросмотра фильтров</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: The new feature uses OpenGL to display stuff, not stuff to display OpenGL. Is it possible that the translation can be (mis)understood as the latter?

File: avidemux_ru.ts [hunk 3]
```diff
@@ -12842,7 +12842,7 @@ Ready.
     <message>
         <location line="+1"/>
         <source>swScale Resizer.</source>
-        <translation>Изменить размер посредством libswscale.</translation>
+        <translation>Изменить размер с помощью swScale.</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: I admit that I didn't dare to fix the original source, but the library is called libswscale while functions from this library are often prefixed with "swscale_" or "sws_". "swScale" is not a thing.

File: avidemux_ru.ts (hunk 1 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Reverted the change

File: avidemux_ru.ts (hunk 2 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Reworded for better meaning.

File: avidemux_ru.ts (hunk 3 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Fixed.

**Comment by eumagga0x2a** [comment]
Thank you.

**Pull request merged**